src/exportedDbTypes.ts

```diff
@@ -1,3 +1,12 @@
 import { Database } from "./database.types";
 
 export type Project = Database["public"]["Tables"]["Project"]["Row"];
+
+export class NoSelectedProject {
```

Comment:
I deleted this file in the other PR, perhaps we should move it to a separate `types` file?

Reply:
On a related note then, we probably just want `Project` to be a type alias for a `string`?